package.json

```diff
@@ -5,7 +5,11 @@
   "scripts": {
     "start": "node devServer.js",
     "build": "webpack",
-    "serve": "ws --spa index.html -d ./dist"
+    "serve": "ws --spa index.html -d ./dist",
+    "tslint": "tslint -p ./tsconfig.json --fix",
+    "tslint:no-fix": "tslint -p ./tsconfig.json",
```

Comment:
Let's call this 'lint' instead of 'tslint:no-fix' to separate *what* we are doing from *how* we are doing it.

Reply:
ok, it's ok for you ```tslint``` and ```lint``` ?
If we take this commands we need to create a task to review this in all projects.

Reply:
yes, both are fine for now. We will remove tslint in a future PR. 
Small steps :-)

Reply:
ok, it's only reporting now

Reply:
Did you push a change?